tests/cpu/Config_tests.cpp

```diff
@@ -7338,7 +7349,10 @@ OCIO_ADD_TEST(Config, add_remove_display)
 
     // Add a (display, view) pair.
 
+    // todo (annie): just a note here that this config will have two (display, view) pairs): sRGB and disp1
```

Comment:
You could remove the todo and make this a comment if you feel it clarifies the test.